include/RAJA/util/View.hpp

```diff
@@ -68,7 +68,14 @@ struct View {
   {
   }
 
-  RAJA_INLINE constexpr View(View const &) = default;
+  //We found that the generated copy constructor does not actually copy-construct
```

Comment:
I would change "the generated ..." to "the compiler-generated..." here with another change noted below.